lib/swiftype-app-search/request.rb

```diff
@@ -68,7 +68,7 @@ def request(method, path, params = {})
         when Net::HTTPForbidden
           raise SwiftypeAppSearch::Forbidden, response_json
         else
-          raise SwiftypeAppSearch::UnexpectedHTTPException, response
+          raise SwiftypeAppSearch::UnexpectedHTTPException.new(response, response_json)
```

Comment:
Could you please add a spec for this case?

Reply:
Absolutely! Sorry for the late reply - I've been down and out with a flu. I'll try to get to this when I'm feeling a bit better. :)